src/hashtable.py

```diff
@@ -51,9 +51,20 @@ def insert(self, key, value):
 
         Fill this in.
         '''
-        pass
+        key_idx = self._hash_mod(key)
+
+        if self.storage[key_idx] == None:
```

Comment:
If the key is not in the hashtable, it will throw an error. Maybe adding one additional condition for the existence of the key can be added to prevent that.